PrismSampleApp/PrismSampleApp/Services/Interfaces/IWebApiService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using PrismSampleApp.Model;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+
+namespace PrismSampleApp.Services.Interfaces
+{
+    public interface IWebApiService
```

Comment:
Instead of IWebApiService, it can be named as IRandomUserService. Always use the name of API whenever you write a service class. Change it in implementation too.

Reply:
Renamed as IRandomUserService